test/testinput/bufr_ncep_highRes_sonde.yaml

```diff
@@ -0,0 +1,209 @@
+              minute: "*/MINU"
+          reportIdentifier:
+            query: "*/RPID"
+          latitude:
+            query: "*/CLATH"
+          longitude:
+            query: "*/CLONH"
+          stationElevation:
+            query: "*/HSMSL"
+          stationIdentification:
+            query: "[*/IDLSIPTM/SMID, NC002104/SMID]"
+          cloudType:
+            query: "[*/CLINRVSD/CLTP[1], */CLINRVSD/CLTP[2], */CLINRVSD/CLTP[3]]"
+
+          # ObsValue
+          pressure:
+            query: "*/UARLVB/PRLC"
+          airTemperature:
+            query: "*/UARLVB/UATMP/TMDB"
+          dewpointTemperature:
+            query: "*/UARLVB/UATMP/TMDP"
+          windDirection:
+            query: "*/UARLVB/UAWND/WDIR"
+          windSpeed:
+            query: "*/UARLVB/UAWND/WSPD"
+
+          # ObsValue - ocean
+          waterTemperature:
+            query: "*/SST1"
+
+          # ObsValue - cloud
+          cloudAmount:
+            query: "*/CLINRVSD/CLAM"
+          heightOfBaseOfCloud:
+            query: "*/CLINRVSD/HOCB"
+          verticalSignificanceSurfaceObservations:
+            query: "[*/CLINRVSD/VSSO[1], */CLINRVSD/VSSO[2]]"
+
+          # QualityMarker
+          stationElevationQM:
+            query: "*/QCEVR"
+          pressureQM:
+            query: "*/UARLVB/QMPR"
+          airTemperatureQM:
+            query: "*/UARLVB/UATMP/QMAT"
+          moistureQM:
+            query: "*/UARLVB/UATMP/QMDD"
+          windQM:
+            query: "*/UARLVB/UAWND/QMWN"
+
+
+    ioda:
+      backend: netcdf
+      obsdataout: "../testoutput/gdas.t12z.adpupa_bufr_nc002103.tm00.nc"
+
+      dimensions:
+        - name: RadiosondeReportLevelData
+          path: "*/UARLVB"
+        - name: CloudInformation
+          path: "*/CLINRVSD"
+        - name: IdentificationOfLaunchSite
+          path: "*/IDLSIPTM"
+
+      variables:
+        # MetaData
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/reportIdentifier"
+          coordinates: "longitude latitude"
+          source: variables/reportIdentifier
+          longName: "Report Identifier"
+
+        - name: "MetaData/latitude"
+          coordinates: "longitude latitude"
+          source: variables/latitude
+          longName: "Latitude"
+          units: "degree_north"
+          range: [-90, 90]
+
+        - name: "MetaData/longitude"
+          coordinates: "longitude latitude"
+          source: variables/longitude
+          longName: "Longitude"
+          units: "degree_east"
+          range: [-180, 180]
+
+        - name: "MetaData/stationElevation"
+          coordinates: "longitude latitude"
+          source: variables/stationElevation
+          longName: "Elevation of Observing Location"
+          units: "m"
+
+        #- name: "MetaData/stationIdentification"
```

Comment:
Probably best to take out commented variables.

Reply:
@nicholasesposito done, removed commented variables from YAML. Thanks